core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -3094,14 +3094,14 @@ protected Scope visitUpdate(Update update, Optional<Scope> scope)
                     .collect(toImmutableMap(ColumnSchema::getName, Function.identity()));
```

Comment:
> Fix column names case sensitivity in UPDATE

As I understand you fix the bug, however it is not clear from the message above. Also there is no github issue for the bug defined. So I think it would be nice to state a little more the problem you are solving. Also you may want to consider to add a test in preparatory commit, so it is clear what behavior you have fixed.